test/jdk/java/awt/Mixing/AWT_Mixing/GlassPaneOverlappingTestBase.java

```diff
@@ -106,14 +116,29 @@ protected boolean performTest() {
         if (!super.performTest()) {
             return false;
         }
+        final CountDownLatch latch = new CountDownLatch(1);
+        f.addFocusListener(new FocusAdapter() {
+            @Override public void focusGained(FocusEvent e) {
+                latch.countDown();
+            }
+        });
         if (testResize) {
             wasLWClicked = false;
             try {
                 SwingUtilities.invokeAndWait(new Runnable() {
 
                     public void run() {
-                        testedComponent.setBounds(0, 0, testedComponent.getPreferredSize().width, testedComponent.getPreferredSize().height + 20);
-                        ancestorLoc = f.getLocationOnScreen();
+                        testedComponent.setBounds(0, 0,
+                                testedComponent.getPreferredSize().width,
+                                testedComponent.getPreferredSize().height + 20);
+                        Component focusOwner = FocusManager.getCurrentManager()
+                                .getFocusOwner();
```

Comment:
Usually we use `KeyboardFocusManager.getCurrentKeyboardFocusManager().getFocusOwner()`